charts/flyte-core/values-onprem.yaml

```diff
@@ -0,0 +1,841 @@
+# ------------------------------------------------------
+#
+# STORAGE SETTINGS
+#
+
+storage:
+  # -- Sets the storage type. Supported values are sandbox, s3, gcs and custom.
+  type: s3
+  # -- bucketName defines the storage bucket flyte will use. Required for all types except for sandbox.
+  bucketName: flytedata
+  # -- settings for storage type s3
+  s3:
+    region: us-east-1
+    endpoint: ""
+    # -- type of authentication to use for S3 buckets, can either be iam or accesskey
+    authType: accesskey
+    # -- AWS IAM user access key ID to use for S3 bucket auth, only used if authType is set to accesskey
+    accessKey: ""
+    # -- AWS IAM user secret access key to use for S3 bucket auth, only used if authType is set to accesskey
+    secretKey: ""
+  # -- settings for storage type gcs
+  gcs:
+  # -- GCP project ID. Required for storage type gcs.
+  # projectId:
+  # -- Settings for storage type custom. See https://github:com/graymeta/stow for supported storage providers/settings.
+  custom: {}
+  # -- toggles multi-container storage config
+  enableMultiContainer: false
+  # -- default limits being applied to storage config
+  limits:
+    maxDownloadMBs: 10
+
+# Database configuration(These are the values for a pgdb instance with hostname of postgres-flyte and postgres/password creds)
+db:
+  datacatalog:
+    database:
+      port: 5432
+      username: postgres
+      password: password
+      host: postgres-flyte
+      dbname: "datacatalog"
+  admin:
+    database:
+      port: 5432
+      username: postgres
+      password: password
+      host: postgresflyte
+      dbname: "flyteadmin"
+# --------------------------------------------------------------------
+# Specializing your deployment using configuration
+# -------------------------------------------------------------------
+#
+# CONFIGMAPS SETTINGS
+#
+
+configmap:
+  clusters:
+    labelClusterMap: {}
+#  labelClusterMap:
+#    team1:
+#      - id: testcluster
+#        weight: 1
+#    team2:
+#      - id: testcluster2
+#        weight: 0.5
+#      - id: testcluster3
+#        weight: 0.5
+    clusterConfigs: []
+#  clusterConfigs:
+#    - name: "testcluster"
+#      endpoint: "testcluster_endpoint"
+#      auth:
+#        type: "file_path"
+#        tokenPath: "/path/to/testcluster/token"
+#        certPath: "/path/to/testcluster/cert"
+#    - name: "testcluster2"
+#      endpoint: "testcluster2_endpoint"
+#      enabled: true
+#      auth:
+#        type: "file_path"
+#        tokenPath: "/path/to/testcluster2/token"
+#        certPath: "/path/to/testcluster2/cert"
+#    - name: "testcluster3"
+#      endpoint: "testcluster3_endpoint"
+#      enabled: true
+#      auth:
+#        type: "file_path"
+#        tokenPath: "/path/to/testcluster3/token"
+#        certPath: "/path/to/testcluster3/cert"
+
+  # -- Configuration for Flyte console UI
+  console:
+    BASE_URL: /console
+    CONFIG_DIR: /etc/flyte/config
+  
+  logger:
+    show-source: true
+    level: 6
```

Comment:
This config seems to be duplicated here https://github.com/flyteorg/flyte/pull/2746/files#diff-20cdbbc75e0c9937a9f82aa3318444cc58ede71498cb7c3ad67f44a42b38aec1R681